testing/opensmtpd-extras/APKBUILD

```diff
@@ -0,0 +1,134 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=opensmtpd-extras
+pkgver=201703132115
+pkgrel=0
+pkgdesc="OpenSMTPD addons"
+url="https://opensmtpd.org/"
+arch="all"
+license="ISC"
+depends="libevent"
+depends_filter_monkey="libressl"
+depends_queue_python="python2"
+depends_scheduler_python="python2"
+depends_table_mysql="mariadb-client-libs"
+depends_table_postgres="libpq"
+depends_table_python="python2"
+depends_table_redis="hiredis"
+depends_table_sqlite="sqlite-libs"
+makedepends="libressl-dev libevent-dev"
+makedepends_queue_python="python2-dev"
+makedepends_scheduler_python="python2-dev"
+makedepends_table_mysql="mariadb-dev"
+makedepends_table_postgres="postgresql-dev"
+makedepends_table_python="python2"
+makedepends_table_redis="hiredis-dev"
+makedepends_table_sqlite="sqlite-dev"
+subpackages=""
+source="https://www.opensmtpd.org/archives/opensmtpd-extras-$pkgver.tar.gz
+	remove-decls.patch"
+builddir="$srcdir/opensmtpd-extras-$pkgver"
+
+_extras="
+	filter-monkey
+	filter-stub
+	filter-trace
+	filter-void
+	queue-null
+	queue-python
+	queue-ram
+	queue-stub
+	scheduler-python
+	scheduler-ram
+	scheduler-stub
+	table-ldap
+	table-mysql
+	table-passwd
+	table-postgres
+	table-python
+	table-redis
+	table-socketmap
+	table-sqlite
+	table-stub
+"
+_extradocs="
+	filter-monkey
+	filter-stub
+	filter-trace
+	filter-void
+	table-sqlite
+	table-redis
+	table-socketmap
+	table-passwd
+"
+
+for _extra in $_extras; do
+	_extraident="${_extra/-/_}"
+	_extramakedeps="$(eval "echo \$makedepends_$_extraident")"
```

Comment:
It’s unnecessary to declare yet another global variable `_extraident` just to use it only one line below. You can do:

```diff
- _extramakedeps="$(eval "echo \$makedepends_$_extraident")"
+ _extramakedeps="$(eval "echo \$makedepends_${_extra/-/_}")"
```

Also you probably want `${_extra//-/_}` (replace all occurrences of `-`, not just the first one).